packages/desktop-client/src/components/reports/ReportOptions.ts

```diff
@@ -7,8 +7,8 @@ import {
   type PayeeEntity,
 } from 'loot-core/src/types/models';
 
-const startDate = monthUtils.subMonths(monthUtils.currentMonth(), 5);
-const endDate = monthUtils.currentMonth();
+const startDate = monthUtils.subMonths(monthUtils.currentMonth(), 5) + '-01';
+const endDate = monthUtils.currentMonth() + '-01';
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏would it make sense to use `currentDate` here instead? WDYT?